fullscreen-api/README.md

```diff
@@ -0,0 +1,6 @@
+
+# Fullscreen API example
+
+The example toggles the video in and out of fullscreen mode on the press of ENTER key.
```

Comment:
Rephrase to "This example allows you to toggle the video in and out of fullscreen mode by pessing the <kbd>Enter</kbd> key."